### PR TITLE
add instructor note, closes #870

### LINKED ISSUE #870: Add an instructor note about RStudio previewing colors
### How could the content be improved?

RStudio has a feature to preview named and hexadecimal colors. This can distract learners (and instructors) who do not expect it.

### Specific Example
The episode that introduces data structures - https://swcarpentry.github.io/r-novice-gapminder/04-data-structures-part1.html

has this code snippet
```r
cats <- data.frame(coat = c("calico", "black", "tabby"),
                    weight = c(2.1, 5.0, 3.2),
                    likes_string = c(1, 0, 1))
```

Under default settings, RStudio will show the word black with a black background.

### Proposed Fix
Per discussion in the slack - https://carpentries.slack.com/archives/C03LE48AY/p1698848972844049

an instructor note would help prepare instructors on this issue, with information about how to disable the feature if desired.

### Which part of the content does your suggestion apply to?

https://swcarpentry.github.io/r-novice-gapminder/04-data-structures-part1.html

### PR DESCRIPTION
**Pull request by ha0ye**
_If this pull request addresses an open issue on the repository, please add 'Closes #NN' below, where NN is the issue number._

_Please briefly summarise the changes made in the pull request, and the reason(s) for making these changes._

_If any relevant discussions have taken place elsewhere, please provide links to these._

<details>

For more guidance on how to contribute changes to a Carpentries project, please review [the Contributing Guide](CONTRIBUTING.md) and [Code of Conduct](https://docs.carpentries.org/topic_folders/policies/code-of-conduct.html).

Please keep in mind that lesson Maintainers are volunteers and it may be some time before they can respond to your contribution. Although not all contributions can be incorporated into the lesson materials, we appreciate your time and effort to improve the curriculum. If you have any questions about the lesson maintenance process or would like to volunteer your time as a contribution reviewer, please contact The Carpentries Team at team@carpentries.org.

</details>

### CODE REVIEW

File: instructors/instructor-notes.md [hunk 1]
```diff
@@ -69,6 +69,21 @@ To prevent the learners from having to re-enter their password each time they pu
...
+
+Mainly, this is likely to come up during the episode on "Data Structures" with the following code block:
+
+```r
+cats <- data.frame(coat = c("calico", "black", "tabby"),
+                    weight = c(2.1, 5.0, 3.2),
+                    likes_string = c(1, 0, 1))
+```
+
+This option can be turned off an on in the following menu setting:
```
Comment (skanwal) [inline instructors/instructor-notes.md:84]: minor typo here "turned off and **on**"

**Review by skanwal — CHANGES_REQUESTED** [review]
Thanks @ha0ye for suggesting this update.

I was reading through this issue online https://github.com/rstudio/rstudio/issues/13118 - looks like even after updating the global setting in Rstudio, the issue persists for few users. It's a WIP, so I suggest we keep an eye on it before merging the change.

File: instructors/instructor-notes.md (hunk 1 above)
Comment (skanwal) [reply instructors/instructor-notes.md]: updated in [`6dab42c` (#871)](https://github.com/swcarpentry/r-novice-gapminder/pull/871/commits/6dab42cb895d37c4a3663d84c8de20752af690ee)

**Review by skanwal — APPROVED** [review]
The issue has been fixed https://github.com/rstudio/rstudio/issues/13118#issuecomment-1814939652. 

This looks good to me now. I have fixed the typo.
Thoughts @matthieu-bruneaux ?

**Comment by naupaka** [comment]
I can verify that this is now fixed for me. But a useful addition. I am going to merge.

**Pull request merged**